13/main/index.html

```diff
@@ -0,0 +1,1480 @@
+        <h4 class="infoHeader" style="background-image:url('images_bw/advt_thumb.jpg')">Adventure</h4>
+                <div class="content cf">
+                <div class="column-three last">
+                <h1 class="center">Adventure Sports</h1>
+                </div>
+                <span id="ZombieZone_image3"><img alt="advt" src="images/advt.jpg" /></span>
+
+
+                                <br />
+                                <br />
+                                <br />
+Felicity will be hosting Skateboarding Workshop and rappelling. We aim at getting you all geared up for those height with rapelling and slam some of those floors with your skateboard skills. Organized in association with GHAC. Registration onsite, 22, 23, 24 February.
+                                <br />
+                                <br />
+                                <br />
+                                <br />
+                                <br />
+                                <br />
+
+                </div>
+        </div>
+
+
+<!-- EVENTS END -->
+
+	<div class="unit item oneWide oneHigh info about" id="felicity">
+		<h4 class="infoHeader" style="background-image:url('images/fel_thumb.jpg')">Felicity</h4>
+		<div class="content cf">
+			<div class="column-three last">
+			<h1>What's Felicity?</h1>
+			<p class="tagline">22,23,24 February'13.</p>
+			</div>
+			<div class="column-three">
+                        <img alt="rangoli" src="images/fel_rangoli.jpg"/>
+				<p> International Institute of Information Technology, Hyderabad is delighted to announce the 12th edition of our annual techno-cultural fest, "Felicity 2013" to be held in February, 22nd-24th.
+This year we hope to awaken the inner child through a language truly remarkable in its simplicity, which is easy to understand and knows no bounds in expression, a universal language with fans in every age-group. This language - “Animatia” - will be used to create a pseudo-reality where people can enjoy three days of unadulterated fun and freedom. </p>
+			</div>
+			<div class="column last">
+			</div>
+
+		</div>
+	</div>
+
+
+
+	<div class="unit item oneWide oneHigh info about" id="contact">
+		<h4 class="infoHeader" style="background-image:url('images/contact_thumb.jpg')">Contact</h4>
+		<div class="content cf">
+			<div class="column-three last">
+			<h1>Contact us</h1>
+			<p class="tagline">We'll be glad to hear from you.</p>
+			</div>
+			<div class="column-two">
+                                <ul>
+                                    <li> Prasanth Garapati : +91 93 97 97 97 49 </li>
+                                    <li> Nishith Maheshwari : +91 77 99 08 77 31 </li>
+                                    <li> SP Mohanty : +91 81 42 09 26 36 </li>
+                                </ul>
+                                <p>
+                                     Official Channel : felicity [at] students [dot] iiit [dot] ac [dot] in
```

Comment:
@polybuildr 
@nisargjhaveri 
@hharchani 

Add real contacts? 

Add warnings that this is no longer real page?